crucible-llvm-syntax/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # next
 
-Nothing yet.
+* Add overrides for string-manipulation `{read,write}-bytes` and
```

Comment:
Perhaps add a colon or other form of punctuation to make this sentence flow a little more nicely, e.g.,

> * Add overrides for string-manipulation: `{read,write}-bytes` and `{read,write}-c-string`.